controllers/postgresqluser_controller.go

```diff
@@ -98,26 +115,69 @@ func (r *PostgreSQLUserReconciler) reconcile(reqLogger logr.Logger, request reco
 		return ctrl.Result{}, err
 	}
 
-	// User instance created or updated
-	reqLogger = reqLogger.WithValues("user", user.Spec.Name, "rolePrefix", r.RolePrefix)
-	reqLogger.Info("Reconciling found PostgreSQLUser resource", "user", user.Spec.Name)
```

Comment:
Any reason for removing this log? I think it still provides value as it shows the controller started to make a decision on a given resource?

Reply:
no that was an accident!